common/transform/src/test/java/ai/h2o/mojos/deploy/common/transform/ScoreRequestTransformerTest.java

```diff
@@ -0,0 +1,80 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import ai.h2o.mojos.deploy.common.rest.model.DataField;
+import ai.h2o.mojos.deploy.common.rest.model.Row;
+import ai.h2o.mojos.deploy.common.rest.model.ScoreRequest;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ScoreRequestTransformerTest {
```

Comment:
Considering that there is a case wherein the field is not present. Should we have a test case covering that scenario as well.

Reply:
good idea ! updated https://github.com/h2oai/dai-deployment-templates/pull/327/commits/844043a958779dce72c770bd3d8e72d72ea41959